test/maps/lua_testsuite.wmf/scripting/militarysite.lua

```diff
@@ -60,6 +60,13 @@ function militarysite_tests:test_set_soldiers_add_and_remove()
    assert_equal(0, self.fortress:get_soldiers({0,0,0,0}))
    assert_equal(0, self.fortress:get_soldiers({3,0,0,1}))
    assert_equal(1, self.fortress:get_soldiers({3,2,0,1}))
+   -- correct type of bob.descr (inclusive attribute)
+   local soldier_bobs = self.fortress.fields[1].bobs
```

Comment:
This line fails 3 times in one testsuite.
Once it returns 695 ([L951](https://github.com/widelands/widelands/actions/runs/5469791867/jobs/9959842054#step:7:951))
and twice 115 ([L3405](https://github.com/widelands/widelands/actions/runs/5469791867/jobs/9959842054#step:7:3405) and [L3782](https://github.com/widelands/widelands/actions/runs/5469791867/jobs/9959842054#step:7:3782))
Will not be easy to find one specific soldier among all the bobs. :confused: 

And [test_road_crosses_another](https://github.com/widelands/widelands/actions/runs/5469791867/jobs/9959843036#step:7:3526) still fails !???